core/tests/test_utils.py

```diff
@@ -3524,6 +3525,44 @@ def publish_collection(self, owner_id: str, collection_id: str) -> None:
         committer = user_services.get_user_actions_info(owner_id)
         rights_manager.publish_collection(committer, collection_id)
 
+    def link_explorations_to_story(
```

Comment:
I think you can call this add_explorations_to_story() -- "link" somehow feels more indirect.

Reply:
Done.